src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -1230,7 +1231,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       <Output
           TaskParameter="TargetOutputs"
           PropertyName="_RestoreFallbackFoldersOverride" />
-    </MsBuild>
+    </MSBuild>
```

Comment:
Similar feedback as above, I think we should revert this.

Reply:
reasons commented below